tests/test_employee_list_page.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+import time
+from libUtils.seleniumUtils.driver_manager import DriverManager
+from webapps.orangeHRM.employeel_list_page import EmployeeListPage
+
+@pytest.fixture
+def setup():
+    driver_manager = DriverManager()
+    yield driver_manager
+    driver_manager.close_session()
+
+def test_search(setup):
```

Comment:
The test function name should be meaningful and it should be reflected test scenarios which are testing 

instead of test_serach , you can define function name as _test_reset_button_check()_

Reply:
resolved